plugins/woocommerce/changelog/44411-e2e-can-edit-product-review

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Add e2e tests to edit and delete product reviews.
```

Comment:
Minor, but there should be a new line at the end of the file.

Reply:
Interesting, I used the template on PR creation. It should be fixed now.

Reply:
Oh, I see. Then we should probably update the tool that automatically creates the changelog. I'll check it out.